src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java

```diff
@@ -185,6 +199,89 @@ private void applyVscConverterStationModification(Network network, Reporter subR
         });
     }
 
+    private void applyShuntCompensatorModification(Network network, Reporter subReporter) {
```

Comment:
The algo is a bit hard to read and has some duplicate lines. I propose you below a shortest version, strictly identical.